### ISSUE TITLE
Default value for parameter + nullable return type hint breaks Squiz.ControlStructures.InlineIfDeclaration

### ISSUE THREAD

**Issue by konecnyjakub** [squizlabs/PHP_CodeSniffer#1466]
This code:

```php
function getUserOrder(int $uid = NULL): ?OrderEntity {
}
```

results in error _Inline shorthand IF statement must be declared on a single line_ when sniff _Squiz.ControlStructures.InlineIfDeclaration_ is used. There should be no error. Happens with version 2.9 and 3.0 of CodeSniffer.

**Comment by jrfnl** [comment]
I think this may be the same issue as already fixed via #1442. The fix is expected to be in the next release.

**Comment by konecnyjakub** [comment]
Yes, it is fixed by that (just tested it). Thank you for a quick response :).
